imjoy_elfinder/views.py

```diff
@@ -69,51 +71,43 @@ def connector(request: Request) -> Response:
             elif field == API_DIRS:
                 http_request[field] = get_all(form, field)
 
-            elif field == API_UPLOAD_PATH:
-                http_request[field] = get_all(form, field)
-
-            # handle CGI upload
-            elif field == API_UPLOAD:
-                http_request[field] = get_all(form, field)
             else:
                 http_request[field] = get_one(form, field)
-
     # run connector with parameters
     status, header, response = elf.run(http_request)
```

Comment:
This should not run in the event loop since we're doing blocking calls here. So either we schedule this on the executor thread pool or we switch to a sync path operation function for the `connector`.

Reply:
For now, perhaps switch to sync would be easier?

Reply:
Yeah, I've solved this in my test branch, based on this branch.